lib/inets/src/http_server/httpd_logger.erl

```diff
@@ -46,11 +46,11 @@ error_report(Protocol, Reason, #mod{init_data = #init_data{peername = PeerName,
               end,
     case Protocol of
         'HTTP' ->
-            Report1#{transport => transport_type(Type)}; 
+            Report1#{transport => transport_type(Type)};
         _ ->
             Report1
-    end. 
-    
+    end.
+
```

Comment:
This is only indentation whitespace changes!?

Reply:
yes. I tried to smuggle small whitespace fixes.